packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -272,7 +275,6 @@ const styles = {
         background: ${background};
         position: sticky;
         top: 0px;
-        z-index: 100;
```

Comment:
This looks like it was accidentally removed

Reply:
Yes, sorry, too zealous taking out these earlier Put back so template container always on top now.